packages/@webex/plugin-cc/src/types.ts

```diff
@@ -168,104 +164,13 @@ export type Team = {
    *  Name of the Team.
    */
   name: string;
-};
-
-/**
- * Represents AuxCode.
- * @public
- */
-
-export type AuxCode = {
```

Comment:
Where was this moved?

Reply:
I removed as we copied types from agentdesktop which has Entity, equivalant of AuxCode. and Entity is makes more sense than AuxCode.